seq2science/scripts/scRNA-seq-1.0.0-rc.1/data/1col_barcode_384.tab

```diff
@@ -0,0 +1,384 @@
+CGTCTAAT
```

Comment:
is this list universal? else, see comment on MT genes :)

Reply:
No, it is not. It gets overwritten by the MT gene list specified in the seq2science config